admin/tests/unit/controllers/authenticated/certifications/certification/details-test.js

```diff
@@ -0,0 +1,64 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { run } from '@ember/runloop';
+import EmberObject from '@ember/object';
+
+module('Unit | Controller | authenticated/certifications/certification/details', function(hooks) {
+  setupTest(hooks);
+
+  const answer = (result) => {
+    return {
+      skill:'@skill1',
+      challengeId:'rec12345',
+      order:'1',
+      result:'ok',
+      jury:result
+    };
+  };
+
+  const competence = (juryScore, ...result) => {
+    return {
+      name: 'Une compétence',
+      index: '1.1',
+      positionedLevel: 3,
+      positionedScore: 26,
+      obtainedLevel: 3,
+      obtainedScore: 26,
+      answers:[answer(result[0]), answer(result[1]), answer(result[2])],
+      juryScore:juryScore ? 12 : false
+    };
+  };
```

Comment:
Une petit suggestion pour tirer parti du fait que result est un tableau. Je l'ai renommé `results` au passage 😃 
```suggestion
  const competence = (juryScore, ...results) => {
    return {
      name: 'Une compétence',
      index: '1.1',
      positionedLevel: 3,
      positionedScore: 26,
      obtainedLevel: 3,
      obtainedScore: 26,
      answers: results.slice(0, 3).map(result => answer(result)),
      juryScore:juryScore ? 12 : false
    };
  };
```